operator-common/src/test/java/io/strimzi/operator/common/operator/MockCertManager.java

```diff
@@ -299,7 +299,7 @@ private static byte[] loadResource(InputStream is) {
      * @param certFile path to the file which will contain the self signed certificate
      * @param sbj      subject information
      * @param days     certificate duration
-     * @throws IOException
+     * @throws IOException  during writing to file
```

Comment:
You should start with a capital letter. Maybe you can also add a verb?

```suggestion
     * @throws IOException  Thrown when writing to file fails
```